server/include/bge/World.h

```diff
@@ -22,7 +23,8 @@
 struct rayIntersection {
     float t;
     glm::vec3 normal;
-    uint32_t tri;
+    uint32_t tri;       // triangle index
```

Comment:
Rana and I had tri there for debugging purposes a long time ago, but I thought it got deleted. I guess not. Are you using it for anything? If not, let's remove it. It hasn't even meant anything since before bucketing.

Reply:
no not using it. can delete